dapr/conf/helpers.py

```diff
@@ -1,56 +1,183 @@
-from typing import Tuple
-
-
-def parse_endpoint(address: str) -> Tuple[str, str, int]:
-    scheme = "http"
-    fqdn = "localhost"
-    port = 80
-    addr = address
-
-    addr_list = address.split("://")
-
-    if len(addr_list) == 2:
-        # A scheme was explicitly specified
-        scheme = addr_list[0]
-        if scheme == "https":
-            port = 443
-        addr = addr_list[1]
-
-    addr_list = addr.split(":")
-    if len(addr_list) == 2:
-        # A port was explicitly specified
-        if len(addr_list[0]) > 0:
-            fqdn = addr_list[0]
-        # Account for Endpoints of the type http://localhost:3500/v1.0/invoke
-        addr_list = addr_list[1].split("/")
-        port = addr_list[0]     # type: ignore
-    elif len(addr_list) == 1:
-        # No port was specified
-        # Account for Endpoints of the type :3500/v1.0/invoke
-        addr_list = addr_list[0].split("/")
-        fqdn = addr_list[0]
-    else:
-        # IPv6 address
-        addr_list = addr.split("]:")
-        if len(addr_list) == 2:
-            # A port was explicitly specified
-            fqdn = addr_list[0]
-            fqdn = fqdn.replace("[", "")
-
-            addr_list = addr_list[1].split("/")
-            port = addr_list[0]     # type: ignore
-        elif len(addr_list) == 1:
-            # No port was specified
-            addr_list = addr_list[0].split("/")
-            fqdn = addr_list[0]
-            fqdn = fqdn.replace("[", "")
-            fqdn = fqdn.replace("]", "")
+from warnings import warn
+from urllib.parse import urlparse, parse_qs, ParseResult
+
+
+class URIParseConfig:
+    DEFAULT_SCHEME = "dns"
+    DEFAULT_HOSTNAME = "localhost"
+    DEFAULT_PORT = 443
+    DEFAULT_AUTHORITY = ""
+    ACCEPTED_SCHEMES = ["dns", "unix", "unix-abstract", "vsock", "http", "https", "grpc", "grpcs"]
```

Comment:
cc @elena-kolevska 
something's off here,  both `grpc` and `grpcs` are not present in the [naming resolution doc](https://github.com/grpc/grpc/blob/master/doc/naming.md) and are also missing in the go-sdk: https://github.com/dapr/go-sdk/blob/main/client/internal/parse.go#L160

Reply:
Fixed in this PR: https://github.com/dapr/python-sdk/pull/700